tests/tests.go

```diff
@@ -1182,6 +1182,25 @@ func (ctl *controller) upsertDeploymentConfig(deploymentConfigUpsert api.Deploym
 	return deploymentConfig, nil
 }
 
+// disableAutomaticBackup disable the automatic backup of database.
+// nolint
+func (ctl *controller) disableAutomaticBackup(databaseID int) error {
+	backupSetting := api.BackupSettingUpsert{
+		UpdaterID:  api.SystemBotID,
```

Comment:
UpdaterID:  api.SystemBotID,

This is not needed as the server will figure it out.